tests/unit_tests/test_serpent_depcode.py

```diff
@@ -94,6 +127,42 @@ def test_convert_nuclide_code_to_name(serpent_depcode):
     assert serpent_depcode.convert_nuclide_code_to_name('390972') == 'Y97m2'
     assert serpent_depcode.convert_nuclide_code_to_name('491142') == 'In114m2'
 
+    serpent_depcode.zaid_convention = 'mcnp'
+    assert serpent_depcode.convert_nuclide_code_to_name('92235.82c') == 'U235'
+    assert serpent_depcode.convert_nuclide_code_to_name('38088.82c') == 'Sr88'
+    assert serpent_depcode.convert_nuclide_code_to_name('95642.82c') == 'Am242m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('95242.82c') == 'Am242'
+    assert serpent_depcode.convert_nuclide_code_to_name('61548.82c') == 'Pm148m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('20060') == 'He6'
+    assert serpent_depcode.convert_nuclide_code_to_name('110241') == 'Na24m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('170381') == 'Cl38m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('310741') == 'Ga74m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('290702') == 'Cu70m2'
+    assert serpent_depcode.convert_nuclide_code_to_name('250621') == 'Mn62m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('300732') == 'Zn73m2'
+    assert serpent_depcode.convert_nuclide_code_to_name('370981') == 'Rb98m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('390972') == 'Y97m2'
+    assert serpent_depcode.convert_nuclide_code_to_name('491142') == 'In114m2'
+
+    serpent_depcode.zaid_convention = 'nndc'
+    assert serpent_depcode.convert_nuclide_code_to_name('92235.82c') == 'U235'
+    assert serpent_depcode.convert_nuclide_code_to_name('38088.82c') == 'Sr88'
+    assert serpent_depcode.convert_nuclide_code_to_name('95242.82c') == 'Am242m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('95642.82c') == 'Am242'
+    assert serpent_depcode.convert_nuclide_code_to_name('61548.82c') == 'Pm148m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('20060') == 'He6'
+    assert serpent_depcode.convert_nuclide_code_to_name('110241') == 'Na24m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('170381') == 'Cl38m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('310741') == 'Ga74m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('290702') == 'Cu70m2'
+    assert serpent_depcode.convert_nuclide_code_to_name('250621') == 'Mn62m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('300732') == 'Zn73m2'
+    assert serpent_depcode.convert_nuclide_code_to_name('370981') == 'Rb98m1'
+    assert serpent_depcode.convert_nuclide_code_to_name('390972') == 'Y97m2'
+    assert serpent_depcode.convert_nuclide_code_to_name('491142') == 'In114m2'
```

Comment:
This could be cleaner if you created a three lists of `nuclide_codes`, `serpent_names`, `mcnp_names`, and `nndc_names` then iterated through them. It would also be a bit easier to see the differences, but since this is a test it probably isn't necessary.